core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorTestBase.java

```diff
@@ -74,6 +74,11 @@ public void before() throws Exception {
     resetManagerView();
   }
 
+  /**
+   * Resets the manager view configured by the parameters in this base class.
+   *
+   * @throws Exception when error happens during creating temporary folder
```

Comment:
You're right that the method is capable of throwing `IOException`. But I wonder who the intended audience is for the javadoc `@throws` comment.

If you dig into the internals of the third line, it runs 
```java
  private BlockMetadataManager() {
    try {
      StorageTierAssoc storageTierAssoc = new WorkerStorageTierAssoc();
      mAliasToTiers = new HashMap<>(storageTierAssoc.size());
      mTiers = new ArrayList<>(storageTierAssoc.size());
      for (int tierOrdinal = 0; tierOrdinal < storageTierAssoc.size(); tierOrdinal++) {
        StorageTier tier = StorageTier.newStorageTier(storageTierAssoc.getAlias(tierOrdinal));
        mTiers.add(tier);
        mAliasToTiers.put(tier.getTierAlias(), tier);
      }
    } catch (BlockAlreadyExistsException | IOException | WorkerOutOfSpaceException e) {
      throw new RuntimeException(e);
    }
  }
```
As you can see, this may throw all sorts of exceptions, but I don't think we want to include `@throws` javadoc to cover all of them since it doesn't help a reader to understand the test better.